docs/guides/admin/docs/configuration/acl.md

```diff
@@ -1,46 +1,152 @@
+attaches to an episode which is part of a series. Opencast can handle this in multiple ways.
+
+The handling is specified by the merge mode configured in `etc/org.opencastproject.organization-mh_default_org.cfg`. It
+defines the relationship between series and episode access control lists, if both are attached to an event. If only one
+list is attached, its rules are always active. If multiple lists are attached, the following modes define Opencast's
+behavior:
+
+### Merge Mode “override” (Default)
+
+The episode ACL takes precedence over the series ACL. This means that the series ACL will be completely ignored as soon
+as the episode has an ACL, no matter what rules are set in either. This allows users to define general rules for a
+series which can be completely redefined on an episode and which are not influenced by changes later made to the series.
+This is also a very simple rule and thus easy to understand.
+
+Example:
+
+|         |ROLE1 |       |ROLE2 |       |ROLE3 |       |
+|---------|------|-------|------|-------|------|-------|
+|         |*read*|*write*|*read*|*write*|*read*|*write*|
+|*series* |allow |allow  |allow |allow  |      |       |
+|*episode*|      |       |allow |       |allow |       |
+|*active* |      |       |allow |       |allow |       |
+
+### Merge Mode “roles”
+
+Series and episode ACLs are merged based on the roles defined within. If both the series and the episode define a rule
+for a specific role (user or group), the episode's rule takes precedence. Rules for roles defined in one ACL only are
+always part of the resulting active ACL.
+
+Example:
+
+|         |ROLE1 |       |ROLE2 |       |ROLE3 |       |
+|---------|------|-------|------|-------|------|-------|
+|         |*read*|*write*|*read*|*write*|*read*|*write*|
+|*series* |allow |allow  |allow |allow  |      |       |
+|*episode*|      |       |allow |       |allow |       |
+|*active* |allow |allow  |allow |       |allow |       |
+
+### Merge Mode “actions”
+
+ACLs are merged based on the actions (read, write, …) contained within both ACLs. If a rule is specified for a tuple of
+role and action in both ACLs, the rule specified in the episode ACL takes precedence.
 
 Example:
 
-    list.name=ACL.ACTIONS
-    # This list provider allows you to configure custom actions that can be added
-    # to ACLs. The default actions are read and write.
-    # The pattern for adding them is
-    # UI_LABEL=actionId
-    #
-    Upload=myorg_upload
-    Download=myorg_downlaod
+|         |ROLE1 |       |ROLE2 |       |ROLE3 |       |
+|---------|------|-------|------|-------|------|-------|
+|         |*read*|*write*|*read*|*write*|*read*|*write*|
+|*series* |allow |allow  |allow |allow  |      |       |
+|*episode*|      |       |allow |       |allow |       |
+|*active* |allow |allow  |allow |allow  |allow |       |
+
+### Switching Modes
+
+Node that switching modes it not necessarily simple since access control lists are cached at several places. Hence,
```

Comment:
Node - Note
...switching modes **it** not... - ...switching modes **is** not... ?